eva/interfaces/relational/db.py

```diff
@@ -169,6 +199,40 @@ def load(
         stmt = parse_load(table_name, file_regex, format, **kwargs)
         return EVARelation(self._evadb, stmt)
 
+    def drop(self, item_name: str, item_type: str, **kwargs) -> EVARelation:
+        """
+        Drop a table or UDF from the database.
+
+        Args:
+            item_name (str): Name of the table or UDF to be dropped.
```

Comment:
I don't like the variable names. It is fine to create 2 APIs, `drop` and `drop_udf` and be precise.